.travis.yml

```diff
@@ -1,28 +0,0 @@
-language: node_js
-sudo: required
-cache: false
-os:
-  - linux
-node_js:
-  - "12.11.0"
-
-services:
-  - xvfb
-
-addons:
-  chrome: stable
-  firefox: stable
-
-script:
-- echo "versions"
-- openssl version
-- node -v
-- npm install
-- npm run lint
```

Comment:
@benjlevesque shouldnt the linting also be in the github actions commands?

Reply:
oops, yeah I missed that! will add it 👍

Reply:
@pubkey fixed!